packages/status-js/src/client/activityCenter.ts

```diff
@@ -0,0 +1,143 @@
+// todo?: rename to notifications (center?), inbox, or keep same as other platforms
```

Comment:
1. shouldn't we open a discussion with product and other platform teams first?
2. else, rename to
    1. `notificationCenter.ts`, `NotificationCenter`, `NotificationCenter.notifications` and `NotificationCenterLatest`
    2. `notifications.ts`, `Notifications`, `Notifications.notifications` 🤔 and `NotificationsLatest`
    3. `inbox.ts`, `Inbox`, `Inbox.notifications` and `InboxLatest`
    4. `ActivityCenter.activities`

Reply:
The "marking as read" might be viewed at as _taking_ an action **now**. And [new mobile designs](https://discord.com/channels/624205794384281629/626744384565674034/999635519262707712), for example, show replying, declining verification requests, transaction actions (send, purchase), and approving join requests too.

All that closer to what GitHub considers to be an activity (done reviews, committed changes, opened issues, etc.) but different in its logging nature (i.e. record of **already** _taken_ actions).

So I believe Notifications (center) like on iOS or Android, and for its familiarity there, is more suitable name for an UI which notifies users about events that they **can** _take_ an action on.